raiden-dapp/src/components/PlannedUdcWithdrawalInformation.vue

```diff
@@ -15,8 +15,8 @@
       <img
         class="planned-udc-withdrawal-information__status-icon mr-2"
         :src="require(`@/assets/planned-udc-withdrawal/status-icon-${statusIconName}.svg`)"
-        :width="iconSize"
-        :height="iconSize"
+        width="13px"
```

Comment:
why hardcode the image size?

Reply:
Because we don't need to vary it. Before it was a variable since it was a shared values for multiple icons. Now there is only one left, so this is not needed anymore. :shrug: